src/components/LoanCards/BorrowerInfo.vue

```diff
@@ -1,13 +1,22 @@
 <template>
 	<div class="borrower-info-wrapper">
-		<router-link :to="`/lend/${ id }`" class="name">{{ name }}</router-link>
+		<router-link
+			:to="`/lend/${ id }`"
+			class="name"
+			v-kv-track-event="['Lending', 'click-Read more', 'Name', id, 'true']">
```

Comment:
I think this one should be something like 'click-borrower name' so we can distinguish which elements are getting clicked (I'll change the link around the borrower photo to 'click-borrower photo') for the same reason.